pkg/csvparser/writer.go

```diff
@@ -28,15 +44,18 @@ import (
 // the underlying io.Writer.  Any errors that occurred should
 // be checked by calling the Error method.
 type Writer struct {
-	Comma   rune // Field delimiter (set to ',' by NewWriter)
-	UseCRLF bool // True to use \r\n as the line terminator
-	w       *bufio.Writer
+	Comma       rune // Field delimiter (set to ',' by NewWriter)
+	Quote       rune // TODO: add comment
+	QuoteFields bool
```

Comment:
Doc please, I can't figure out what it does.

Will enabling it force quotes on all fields? Is this what S3 does? If so rename to `AlwaysQuote`.

Reply:
> Will enabling it force quotes on all fields? Is this what S3 does?
Yes

> If so rename to `AlwaysQuote`.
Done